spec/lib/vanagon/platform_spec.rb

```diff
@@ -4,22 +4,31 @@
   let(:platforms) do
     [
       {
-        :name         => "debian-6-i386",
-        :os_name      => "debian",
-        :os_version   => "6",
-        :architecture => "i386",
+        :name                   => "debian-6-i386",
+        :os_name                => "debian",
+        :os_version             => "6",
+        :architecture           => "i386",
+        :output_dir             => "debian/6/i386",
+        :output_dir_with_target => "debian/6/thing/i386",
```

Comment:
This was originally in the form `deb/lucid/thing` but this test indicates that we should expect thesecomputed values to transition to a form like `debian/6/thing/i386`; will this require updates to the packaging repo to keep things from blowing up when we try to ship Vanagon packages built from this point on? It's late and I'm fuzzy on the interactions here.